cishouseholds/pipeline/bloods_delta_ETL.py

```diff
@@ -1,18 +1,40 @@
+from pyspark.accumulators import AddingAccumulatorParam
+from pyspark.sql import DataFrame
+
 from cishouseholds.derive import assign_column_uniform_value
 from cishouseholds.derive import substring_column
-
-
-def bloods_delta_ETL():
-    extract_bloods_delta()
-    transform_bloods_delta()
-    load_bloods_delta()
-
-
-def extract_bloods_delta():
-    pass
-
-
-def transform_bloods_delta(df):
+from cishouseholds.extract import read_csv_to_pyspark_df
+from cishouseholds.pipeline.input_variable_names import bloods_variable_name_map
+from cishouseholds.pipeline.validation_schema import bloods_validation_schema
+from cishouseholds.pyspark_utils import convert_cerberus_schema_to_pyspark
+from cishouseholds.pyspark_utils import get_or_create_spark_session
+from cishouseholds.validate import validate_and_filter
+
+# from pyspark.sql import SparkSession
+
+
+def bloods_delta_ETL(delta_file_path: str):
+    spark_session = get_or_create_spark_session()
+    bloods_spark_schema = convert_cerberus_schema_to_pyspark(bloods_validation_schema)
```

Comment:
Will this function be imported to another file or always run in this file? Thinking `bloods_validation_schema` should be a named parameter if so.

Reply:
If it's going to be run in this file only then absolutely fine, disregard the comment. If not - the same goes for `bloods_variable_name_map` (impure function otherwise).

Reply:
Also depends on how this was done for swabs too I suppose

Reply:
I've basically mirror'd the swabs.  I think these functions will only be used in their own files (although could potentially change)